webpages/popup/style.css

```diff
@@ -20,8 +20,15 @@ body {
   display: flex;
   height: 60px;
 }
+.header-button {
+  right: 0;
+  position: fixed;
```

Comment:
```suggestion
```
This isn't necessary because the title has `flex-grow: 1`.

Reply:
It was already there, dunno if it was there for some reason...

Reply:
It's needed on the settings page but not in the popup.

Reply:
oh ok